templates/autoscale-new-vpc.template

```diff
@@ -0,0 +1,1163 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "Fortinet FortiGate Auto Scaling [new VPC]. This template creates a new VPC and deploys Fortinet FortiGate Auto Scaling into it. For more details, please see the Fortinet FortiGate Auto Scaling documentation. **WARNING** You will be billed for the FortiGate instances and related AWS resources if you create a stack from this template.",
+    "Parameters": {
+        "ResourceTagPrefix": {
+            "Type": "String",
+            "AllowedPattern": "[0-9a-zA-Z@.#-]+",
+            "MaxLength": 50,
+            "ConstraintDescription": "Resource tag prefix can include numbers, lowercase letters, uppercase letters, ampersat(@) , hyphens (-), period (.), and hash (#). Max length is 50.",
+            "Description": "The ResourceGroup Tag Key used on all resources and as the name prefix of all applicable resources. Can only contain numbers, lowercase letters, uppercase letters, ampersat(@), hyphens (-), period (.), and hash (#). Max length is 50."
+        },
+        "CustomIdentifier": {
+            "Type": "String",
+            "Default": "fgtASG",
+            "MaxLength": "10",
+            "AllowedPattern": "[A-Za-z0-9]+",
+            "ConstraintDescription": "must only contain uppercase and lowercase letters and numbers",
+            "Description": "An alternative name prefix to be used on a resource that the 'Resource tag prefix' cannot apply to. Can only contain numbers, lowercase letters, and uppercase letters. Max length is 10."
+        },
+        "QSS3BucketName": {
+            "Type": "String",
+            "Default": "aws-quickstart",
+            "AllowedPattern": "^[0-9a-zA-Z]+([0-9a-zA-Z-]*[0-9a-zA-Z])*$",
+            "ConstraintDescription": "Quick Start S3 bucket name can include numbers, lowercase letters, uppercase letters, and hyphens (-). It cannot start or end with a hyphen (-).",
+            "Description": "The name of the S3 bucket you created for your copy of Quick Start assets, if you decide to customize the Quick Start for your own use. It can only contain numbers, lowercase letters, uppercase letters, and hyphens (-), but should not start or end with a hyphen (-)."
```

Comment:
```suggestion
            "Description": "The name of the S3 bucket you created for your copy of Quick Start assets if you decide to customize the Quick Start for your own use. It can only contain numbers, lowercase letters, uppercase letters, and hyphens (-), but should not start or end with a hyphen (-)."
```